tests/settings/merged/__init__.py

```diff
@@ -7,7 +7,7 @@
     'components/apps_middleware.py',
     'components/static.py',
     'components/templates.py',
-    'components/database.py',
+    'components/database.pyc',
```

Comment:
My proposal will prevent possible typos when `.py` is mistyped as `.pyc` (it can happen)